ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -21,16 +22,25 @@
 -- an unexplained thunk in 'KnownIntersectionState' and thus a space leak. See
 -- #1356.
 
-module Ouroboros.Consensus.MiniProtocol.ChainSync.Client (
-    ChainDbView (..)
+-- | This module is intended for qualified import.
```

Comment:
I would use this opportunity to link to the relevant external documentation here, like `typed-protocols` and the mini-protocols document as well as the ouroboros-network implementation of the ChainSync miniprotocol.